components/code-editor/README.md

```diff
@@ -54,3 +54,21 @@ editor. It is passed the new value as an argument.
 
 - Type: `Function`
 - Required: No
+
+### settings
+
+Used to setup the Code Editor and includes options for linting and CodeMirror. By default this will use `window._wpGutenbergCodeEditorSettings`.
```

Comment:
Let's clarify here that these settings will be passed into `wp.codeEditor.initialize()`, e.g.

```md
### settings

The settings object used to initialize the WordPress code editor. These are passed into `wp.codeEditor.initialize()`. If you do not specify a settings object, `window._wpGutenbergCodeEditorSettings` will be used.
```